pywr-schema/src/nodes/annual_virtual_storage.rs

```diff
@@ -167,15 +159,22 @@ impl TryFrom<AnnualVirtualStorageNodeV1> for AnnualVirtualStorageNode {
             .map(|v| v.try_into_v2_parameter(Some(&meta.name), &mut unnamed_count))
             .transpose()?;
 
+        let initial_volume = if let Some(v) = v1.initial_volume {
+            StorageInitialVolume::Absolute(v)
+        } else if let Some(v) = v1.initial_volume_pc {
+            StorageInitialVolume::Proportional(v)
+        } else {
+            StorageInitialVolume::default()
```

Comment:
Should this return a `Err(ConversionError::MissingAttribute)`? Or does Pywr v1 not require this attribute?